Firestore/Swift/Source/Codable/DocumentID.swift

```diff
@@ -118,17 +118,18 @@ public struct DocumentID<Value: DocumentIDWrappable & Codable>:
     get { value }
     set {
       if let someNewValue = newValue {
-        logWarning(for: someNewValue)
+        logWarning(with: someNewValue)
       }
       value = newValue
     }
   }
 
-  private func logWarning(for value: Value) {
-    FIRLogWarningSwift(
-      "[FirebaseFirestoreSwift]",
-      "I-FST000002",
-      """
+  private func logWarning(with value: Value) {
```

Comment:
nit: I don't feel strongly about it but potentially something like `logIgnoredValueWarning` since that function is for a specific logging case.

Reply:
Thanks, I prefer that. Done in 4feb8fc071d23a004f55d9ad619022d071595e33